cypress/integration/examples/connectors.spec.js

```diff
@@ -0,0 +1,97 @@
+/// <reference types="cypress" />
```

Comment:
Remove the reference types comment, as it is not necessary with the latest versions of Cypress.